grails-app/views/product/_productCatalogs.gsp

```diff
@@ -1,86 +1,85 @@
 <div id="productCatalogs">
-
     <div class="box dialog">
         <h2>
             <warehouse:message code="product.catalogs.label" default="Product Catalogs"/>
         </h2>
-        <table>
-            <thead>
-            <tr>
-                <th>
-                    <warehouse:message code="default.code.label"/>
-                </th>
-                <th>
-                    <warehouse:message code="default.name.label"/>
-                </th>
-                <th>
-                    <warehouse:message code="default.actions.label" default="Actions"/>
-                </th>
-            </tr>
-            </thead>
-            <tbody>
-            <g:each var="productCatalog" in="${productCatalogs }" status="status">
-                <tr class="prop ${status%2?'even':'odd'}">
-                    <td>
-                        <g:link controller="productCatalog" action="edit" id="${productCatalog.id}">
-                            ${productCatalog?.code }
-                        </g:link>
-                    </td>
-                    <td>
-                        <g:link controller="productCatalog" action="edit" id="${productCatalog.id}">
-                            ${productCatalog?.name }
-                        </g:link>
-                    </td>
-                    <td>
-                        <g:remoteLink action="removeFromProductCatalog" id="${product.id}" params="['productCatalog.id':productCatalog.id]"
-                                      class="button" update="productCatalogs">
-                            <g:message code="default.button.delete.label"/>
-                        </g:remoteLink>
-                    </td>
-                </tr>
-            </g:each>
-            <g:unless test="${productCatalogs }">
+        <g:formRemote name="addToProductCatalog"
+                          update="productCatalogs" onSuccess="onSuccess(data,textStatus)" onComplete="onComplete()"
```

Comment:
`onSuccess` does not take any parameters (this applies to every onSuccess on these files)